packages/@aws-cdk/aws-stepfunctions/lib/input.ts

```diff
@@ -53,6 +59,14 @@ export class TaskInput {
  * The type of task input
  */
 export enum InputType {
+  /**
+   * Use a literal string
+   * This might be a JSON-encoded object, or just text.
+   */
   TEXT,
+  /**
+   * Use an object which may contain Data and Context fields
+   * as object values, if desired.
+   */
   OBJECT
```

Comment:
Can we provide a few examples to clarify this?

Reply:
added